src/conditionals/programs_with_decision_points/hard/q13/Bookstore.java

```diff
@@ -0,0 +1,61 @@
+package conditionals.programs_with_decision_points.hard.q13;
+
+// A store sells books based on color and book genre.
+// The user selects a color:
+// (1 for Pink)
+// (2 for Blue)
+// (3 for Green)
+
+// Then, the user selects a book genre:
+// (1 for Fiction)
+// (2 for Non-Fiction)
+// (3 for Mystery)
+
+// Write a program using nested switches to print the selected book.
+// If the user enters an invalid answer, print "Invalid selection."
```

Comment:
This question seems to be Java so you need to include all the exact strings you've used to test it (i.e. Select book color: (1 - Pink, 2 - Blue, 3 - Green)". Easiest way to do so is to include a "Sample Output:" block at the bottom where you print a sample input/output case.

Reply:
It's actually a Parson's question, would I still need to do that?

Reply:
nope